site-new/src/components/mailchimp.tsx

```diff
@@ -1,5 +1,111 @@
-import React from 'react';
+import '@ant-design/v5-patch-for-react-19'; //  Make antd work properly in React 19. See https://ant.design/docs/react/v5-for-19.
+import { MailOutlined } from '@ant-design/icons';
+import { Button, Input, message } from 'antd';
+import { ButtonType } from 'antd/es/button';
+import EmailValidator from 'email-validator';
+import jsonp from 'jsonp';
+import React, { useState, useCallback } from 'react';
+import DOMPurify from 'dompurify';
 
-const Mailchimp: React.FC<any> = () => <div>Mailchimp</div>;
+import Emoji from './emoji';
+import styles from './mailchimp.module.css';
+
+interface MailchimpProps {
+  url?: string;
+  botCode?: string;
+  buttonType?: ButtonType;
+}
+
+const Mailchimp: React.FC<MailchimpProps> = (givenProps) => {
```

Comment:
Question) I wasn't sure if the mail subscription worked. Occasionally, I show error messages. 

<img width="824" height="381" alt="Image" src="https://github.com/user-attachments/assets/787055dc-e622-42e0-96c3-b3e791dca66e" />

Reply:
As I reported in the Slack channel, the mail subscription does not work on the armeria.dev, too. Our MailChimp is not working correctly. I will test and make sure it works, once the MailChimp is restored to normal.

For now, usually it fails without an error message. I suppose `jsonp` is not able to handle the error correctly because of the way it works. (If we could work around the CORS in any other way, we may consider using a different library.)
For the error you've encountered, I added a commit to handle that error case: [f6f605f](https://github.com/line/armeria/pull/6306/commits/f6f605f4dc758abf522a396858bd73faad13cb26)